src/IBM.Watson.Assistant.v1/Model/RuntimeEntityAlternative.cs

```diff
@@ -0,0 +1,39 @@
+/**
+* (C) Copyright IBM Corp. 2018, 2020.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+
+using Newtonsoft.Json;
+
+namespace IBM.Watson.Assistant.v1.Model
```

Comment:
Noth a big deal, well no even a deal. Just out of curiosity, why did we go with `v1` instead of `V1`.